xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -64,7 +64,7 @@
  
  * The implementation of <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)?displayProperty=nameWithType> can add new dependencies which may need to be isolated to allow different versions to exist. The most natural implementation would place these dependencies in the default context. Careful design can isolate the new dependencies.
  
- * The binding isolation process creates multiple types with the same type name.
+ * The same assembly is loaded multiple times into different contexts.
      * This can lead to confusing error messages, for example "type Foo.Foo cannot be cast to type Foo.Foo."
```

Comment:
Can you please also replace "Foo" with something more specific - if I remember correctly "Foo" is now not recommended for "arbitrary function names" anymore.